cymetric/convenient_interface.py

```diff
@@ -0,0 +1,650 @@
+#!/usr/bin/env python
+
+import warnings
+
+import pandas as pd
+import numpy as np
+
+import cymetric as cym
+from cymetric import tools
+
+try:
+    from pyne import data
+    import pyne.enrichment as enr
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
+
+try:
+    from graphviz import Digraph
+    HAVE_GRAPHVIZ = True
+except ImportError:
+    HAVE_GRAPHVIZ = False
+
+
+def format_nuclist(nuc_list):
+    """
+    format the nuclide list provided by the users into a standart format:
+    ZZAAIIIII.
+    Parameters
+    ----------
+    nuc_list:  list of nuclides
```

Comment:
It is generally advisable to not include the type in the variable name.  Instead use the plural, `nucs`.